lib/stack_master/commands/compile.rb

```diff
@@ -5,7 +5,7 @@ class Compile
       include Commander::UI
 
       def perform
-        puts(proposed_stack.template_body)
+        StackMaster.stdout.puts(proposed_stack.template_body)
```

Comment:
Print the compiled template to the StackMaster configured STDOUT.

This allows the printed content to be captured during Cucumber tests, so we can write assertions for it.